RecoBTag/PixelCluster/plugins/PixelClusterTagInfoProducer.cc

```diff
@@ -22,7 +22,89 @@
 //
 //
 
-#include "RecoBTag/PixelCluster/interface/PixelClusterTagInfoProducer.h"
+#ifndef RecoBTag_PixelCluster_PixelClusterTagInfoProducer_h
+#define RecoBTag_PixelCluster_PixelClusterTagInfoProducer_h
```

Comment:
L25 and L26 not needed, since this file is not to be included anywhere

Reply:
Ok, done!